examples/example_multi_genome.py

```diff
@@ -0,0 +1,138 @@
+"""
+Example for evolutionary regression with multiple genomes
+=========================================================
+
+Example demonstrating the use of Cartesian genetic programming with multiple
+genomes per individual for a simple regression task with a piecewise-defined
```

Comment:
```suggestion
genomes per individual for a simple regression task with a piecewise
```
but this is a matter of taste ;)